docs/website/developers/committers/release-process/prerequisites.md

```diff
@@ -32,16 +32,24 @@ pre-release software to the global release mirror network!
 GPG keys
 --------
 
-The release manager must have a GPG key to be used to sign the release.
+The release manager must have a GPG key to be used to sign the release. See below to install `gpg2`
+(with a `gpg` alias).  The steps here also assume you have the following set
+(not using `whoami` if that's not appropriate):
+
+{% highlight bash %}
+GPG_KEY=`whoami`@apache.org
+SVN_USERNAME=`whoami`
+{% endhighlight %}
```

Comment:
They're only ever needed in this section, which is why I think they're better here.